Posters/Three Column Landscape/Example/Poster.tex

```diff
@@ -0,0 +1,155 @@
+% Poster Template for the Okinawa Institute of Science and Technology (OIST)
+% Created by Gaston Sivori in May 2021 based on Two-columns poster template by Jeremie Gillet (June 2019)
+
+\documentclass[final]{beamer}
+
+\mode<presentation>{\usetheme{OIST}}
+
+% Packages needed for the poster
+\usepackage[orientation=landscape,size=a0,scale=1.4]{beamerposter}
+\usepackage[english]{babel}
+\usepackage[latin1]{inputenc}
+
+% Style of equations
+\usefonttheme[onlymath]{serif}
+\boldmath
+
+% Put images in the figures folder
+\graphicspath{{figures/}}
+
+% Add packages that you need
+\usepackage{amsmath}
+\usepackage{lipsum}  % For dummy test
+\usepackage{multicol} % For multiple columns
+\usepackage{ragged2e}
```

Comment:
What is this for? I'm guessing for using `\justify`, is that right?

Reply:
In the example, I make use of some of those packages to have a better fit of the displayed paragraphs. Indeed ragged2e is for justify to have nicer spacing and to justify appropriately. amsmath is for the math typography which is easier to read. lipsum is actually not used but it is used in the template to generate lorem lipsum text. And multicol is to use three-column landscape. Somehow, using multicol{3} and then defining each column by their width given \textwidth seems an easier approach. If the user wants to make two small columns and a big one for figures that's easily down explicitly by giving the width that you want for each column. Could you compile it in the end? Overleaf shows no issues in compilation.

Reply:
More on multicol:
Purpose of the multicol package is to enable the flexible use of multiple columns (up to 9), balance column to equal or near equal height and width, and support changing the number of columns on a single page (requires balancing).
You will notice that if you use columns[t] and define columns the way I did, the moment you want to start messing with their widths it becomes harder to make LaTex to do what you want it to do.
In general I prefer multicol, but if you think the template can be used without it, go ahead with changing it!